File: tests/send_invite.php
<?php
//$url = 'http://server.com/path';
$url = 'http://52.8.173.36/metster/send_gcm_message.php';
$param1 = '859834507580812';
$param2 = '859834507580812';
$event_ref = '859842507380812-->event-->0';
$name = 'naveen';

$payload = array('host' => $param1, 'to_id' => $param2, 'payload_type' => 'invite_check', 'payload_message' => 'sounds_great', 'event_refrence' => $event_ref, 'sender_name' => $name );

$param3 = json_encode($payload);
$data = array('param1' => $param1, 'param2' => $param2, 'param3' =>  $param3);

// use key 'http' even if you send the request to https://...
$options = array(
    'http' => array(
        'header'  => "Content-type: application/x-www-form-urlencoded\r\n",
        'method'  => 'POST',
        'content' => http_build_query($data),
    ),
);
$context  = stream_context_create($options);
$result = file_get_contents($url, false, $context);

var_dump($result);
?>

File: tests/send_invite.py
import urllib, urllib2
import json
import random

url = 'http://52.8.173.36/metster/send_gcm_message.php'
names = ["allie","david","jinfeng","mike","chris","kamayaki"]
event_name = ["dinner", "birthday", "party tonight"]
param1 = "812345607380812"
param2 = "859842507380812"
event =  "812345607380812-->event-->0"

elon = "865432107380812"
alan = "812345607380812"

payload = dict()
payload['host'] = elon # elon invite
payload['to_id'] = "859842507380812"
payload['payload_type'] = "invite_check"
payload['payload_message'] = "dinner?"
payload['sender_name'] = "elon musk"
payload['event_reference'] =  elon + "-->event-->0"
payload['event_name'] = "lunch"

jdata = json.dumps(payload)

values = {'param1' : param1,
          'param2' : param2,
          'param3' : jdata }


data = urllib.urlencode(values)
req = urllib2.Request(url, data)
response = urllib2.urlopen(req)
print response.geturl()
print response.info()
the_page = response.read()
print the_page
